host/xtest/acipher_perf.c

```diff
@@ -661,7 +661,7 @@ static int check_rsa_hash_params(uint32_t rsa_algo, int width_bits, int size,
 	    width_bytes < hash_len + DERCODE_SHA1_LEN + PKCS_V1_5_MIN) {
 		fprintf(stderr, "The size or algo is not valid!\n");
 		return -1;
-	} else if (rsa_algo != RSASSA_PKCS1_V1_5_SHA1 &&
+	} else if (rsa_algo >= RSASSA_PKCS1_V1_5_SHA1 &&
```

Comment:
I don't know what others think but I prefer switch/case for this kind of test, because arithmetic comparison assume a precise order and it is more difficult to review and make sure all cases are covered properly.